src/comment.rs

```diff
@@ -1075,8 +1074,7 @@ fn light_rewrite_comment(
             // Preserve markdown's double-space line break syntax in doc comment.
             trim_end_unless_two_whitespaces(left_trimmed, is_doc_comment)
         })
-        .collect();
-    lines.join(&format!("\n{}", offset.to_string(config)))
+        .join(&format!("\n{}", offset.to_string(config)))
```

Comment:
Okay, so just want to make sure I'm understanding this correctly. This avoids allocating the `Vec<&str>` and directly allocates into a `String`?

Reply:
Yes, exactly.